ncm-spma/src/main/pan/components/spma/yumng/schema.pan

```diff
@@ -0,0 +1,73 @@
+# ${license-info}
+# ${developer-info}
+# ${author-info}
+
+declaration template components/spma/yumng/schema;
+
+include 'components/spma/schema';
+
+type SOFTWARE_GROUP = {
+    "default" : boolean = true
+    "mandatory" : boolean = true
+    "optional" : boolean = false
+    "names" : string[]
```

Comment:
Me too. :( I think the problem was that the removal of the {} on the line below was so subtle it didn't get caught in code review. :(